src/client/components/ContactForm/ContactForm.component.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './ContactForm.styles.css';
+
+export default function ContactForm({user}) {
+    return (
+        <div className="infoBox">
+            <div className="wrapper">
+                <form className="form">
+                    <label className="title">CONTACT:</label>
+                    <label className="fullName">Full-name:  {
+                        user.fullName
+                    }</label>
+                    <label className="email">e-mail address:  {
+                        user.email
+                    }</label>
+                </form>
+            </div>
+        </div>
+    );
+}
+
+ContactForm.defaultProps = {
+    user: {
+        fullName: "",
+        email: ""
+    }
+};
+
+ContactForm.propTypes = {
+    user: PropTypes.shape(
+        {fullName: PropTypes.string, email: PropTypes.string}
+    )
+};
```

Comment:
Update the proptypes and I would suggest making both properties required as well.

```suggestion
ContactForm.propTypes = {
  email: PropTypes.string.isRequired,
  fullName: PropTypes.string.isRequired,
};
```

Reply:
Made changes according to the review comments